### ISSUE TITLE
Function After Object restricted by Unexpected Token.

### ISSUE THREAD

**Issue by Rakib-R** [yannickcr/eslint-plugin-react#1486]
IDK what happened to this simple code . Actually I cant pass this after @todo:[]  .Tried this  from Chrome console to ESlint. Take a look .
```js
var todoList = {
  todo : []

addTodo :function(HEY){
this.todo.push({
  textTodo : HEY,
  completed : true
});
 this.displayTodo();
},

 displayTodo :function(){
console.log(this.todo) ;
if (this.todo.length === 0){
  console.log("You're Are Empty !") ;
}else {
   console.log(this.todo[i].textTodo) ;
}
for (var = i ; i<this.todo.length ; i++){
  console.log(this.todo[i].textTodo);
}
if (this.todo[i].completed === true){
console.log("Right");
} else {
   console.log("Wrong");
}
},
changeTodo: function(p,n){
  this.todo[p] = n;
this.displayTodo();
}
};
```

**Comment by jomasti** [comment]
This doesn't seem to be related to this plugin since it's basic Javascript and not using React. Also, you have some syntax errors such as a missing comma after `todo: []` and defining the variable incorrectly in your for loop. I would also suggest using [code blocks](https://help.github.com/articles/creating-and-highlighting-code-blocks) for future issues to make your code samples easier to read.

**Comment by ljharb** [comment]
Indeed, this is invalid JavaScript syntax.

**Comment by Rakib-R** [comment]
Thanks For The Reply
